ui/app/adapters/job.js

```diff
@@ -19,6 +19,15 @@ export default class JobAdapter extends WatchableNamespaceIDs {
     return this.ajax(url, 'GET');
   }
 
+  fetchRawSpecification(job) {
+    const url = addToPath(
+      this.urlForFindRecord(job.get('id'), 'job', null, 'submission'),
+      '',
+      'version=' + job.get('version')
```

Comment:
`version=${job.get('version')}` would help you avoid future concatenation + line return readability woes at some point, but this is fine either way